src/MetabaseTab/MetabaseTab.tsx

```diff
@@ -5,6 +5,25 @@ import { METABASE_PORT, METABASE_SESSION_NAME } from '../constants';
 
 import './MetabaseTab.css';
 
+const CUSTOM_CSS = `
```

Comment:
> Metabase-version-dependent, correct

Yes, it's some sort of unique hash to avoid CSS collisions. This may change without notice so we have to lock versions.